main.tf

```diff
@@ -1,40 +1,60 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+data "aws_secretsmanager_secret" "db_secret" {
+  arn = "arn:aws:secretsmanager:us-west-2:659808933566:secret:db-secret-J1Z9s0"
```

Comment:
has the secret been created manually?
if yes - please use `terraform` to manage it, then `import` the object to tfstate file

Reply:
Done.